src/gui/annotator/AnnotationWidget.cpp

```diff
@@ -190,10 +195,14 @@ void AnnotationWidget::setSettingsCollapsed(bool isCollapsed)
 	}
 }
 
-void AnnotationWidget::showControlsWidget()
+void AnnotationWidget::setControlsWidgetVisible(bool enabled)
 {
-	insertDockWidget(Qt::BottomDockWidgetArea, mControlsWidget);
-	restoreDockWidgets();
+	if (enabled) {
+		restoreDockWidget(mControlsDockWidget);
+		restoreDockWidgets();
```

Comment:
Something needs to be renamed here, probably our own method, one is called `restoreDockWidget` and the other `restoreDockWidgets` that's to similar.

Reply:
Maybe rename `restoreDockWidgets` to `restoreWidgets`, that one is unused in the QT framework. Or `restoreWidgetsState` or `restoreDockWidgetsState`.
Last one is my fav, keeps current and adds `State` that's used inside it with `restoreState`/

Reply:
Yeah, I think `restoreDockWidgetsState` is my favorite too.